input/templates/partials/myaccount/my-address-book-adress.hbs

```diff
@@ -0,0 +1,18 @@
+<div>{{address.title}} {{address.firstName}} {{address.lastName}}</div>
+<div>{{address.streetName}}</div>
+<div>{{address.additionalStreetInfo}}</div>
```

Comment:
These two are tricky, because as we don't have an `address` field in the Address from our platform, in Sunrise we use a combination of `streetName` and `additionalStreetInfo`. Not evident ;)